src/Ninja/G1CP/Content/Symbols/string.d

```diff
@@ -241,3 +241,17 @@ func int G1CP_FindStringConstArrIdxI(var int symbId, var string needle) {
 func int G1CP_FindStringConstArrIdx(var string name, var string needle) {
     return G1CP_FindStringConstArrIdxI(MEM_GetSymbolIndex(name), needle);
 };
+
+/*
+ * Replace element in string array constant
```

Comment:
We should clarify some more details in this header comment. Maybe something similar to the function above  `Find a string in a constant string array (case-sensitive) and replace it if found`.

Reply:
And maybe: `and returns true on success`.